.github/workflows/validate-pr.yml

```diff
@@ -49,14 +49,20 @@ jobs:
     runs-on: ubuntu-20.04
     env:
       PR_TITLE: ${{ github.event.pull_request.title }}
+      TARGET_BRANCH: ${{ github.event.pull_request.base.ref }}
```

Comment:
[There's a predefined env var for that:](https://docs.github.com/en/actions/learn-github-actions/variables#default-environment-variables:~:text=GITHUB_BASE_REF,example%2C%20main.) `$GITHUB_BASE_REF`